### PR TITLE
TSK-369: Fix Sonarcloud with testcoverage

### PR DESCRIPTION
**Pull request by benjamineckstein**
taskana travis needs a valid SONAR_TOKEN to work.

### CODE REVIEW

File: lib/taskana-cdi-example/pom.xml [hunk 1]
```diff
@@ -26,5 +26,23 @@
 			<artifactId>taskana-cdi</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+		<dependency>
```
Comment (mustaphazorgati) [inline lib/taskana-cdi-example/pom.xml]: Please use the aggrigator junit-jupiter

File: lib/taskana-cdi-example/src/test/java/pro/taskana/ExampleBootstrap.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package pro.taskana;
```
Comment (mustaphazorgati) [inline lib/taskana-cdi-example/src/test/java/pro/taskana/ExampleBootstrap.java]: why does this test class exist? Honestly I dont see any tests coming soon for our spring-boot demo

File: pom.xml [hunk 3]
```diff
@@ -75,11 +75,14 @@
...
 		<version.equalsverifier>3.1.11</version.equalsverifier>
 		<version.openpojo>0.8.13</version.openpojo>
+		<version.jacoco>0.8.3</version.jacoco>
 
 		<!-- database driver versions -->
 		<version.h2>1.4.197</version.h2>
 		<version.db2>11.1.1.1</version.db2>
 		<version.postgres>42.2.9</version.postgres>
+		<!-- used by jacoco to collect coverage -->
+		<argLine></argLine>
```
Comment (mustaphazorgati) [inline pom.xml]: Can you explan this a little further or link me a referrence? I don't see why an empty property will help

File: rest/taskana-rest-spring-example-boot/pom.xml [hunk 4]
```diff
@@ -95,8 +95,25 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
```
Comment (mustaphazorgati) [inline rest/taskana-rest-spring-example-boot/pom.xml]: Please use the aggrigator junit-jupiter

File: rest/taskana-rest-spring-example-boot/src/test/pro/taskana/rest/ExampleRestApplicationTest.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
+package pro.taskana.rest;
```
Comment (mustaphazorgati) [inline rest/taskana-rest-spring-example-boot/src/test/pro/taskana/rest/ExampleRestApplicationTest.java]: why does this test class exist? Honestly I dont see any tests coming soon for this module

File: lib/taskana-cdi-example/pom.xml (hunk 1 above)
Comment (benjamineckstein) [reply lib/taskana-cdi-example/pom.xml]: done

File: rest/taskana-rest-spring-example-boot/src/test/pro/taskana/rest/ExampleRestApplicationTest.java (hunk 5 above)
Comment (benjamineckstein) [reply rest/taskana-rest-spring-example-boot/src/test/pro/taskana/rest/ExampleRestApplicationTest.java]: removed test classes and test dependencies

File: rest/taskana-rest-spring-example-boot/pom.xml (hunk 4 above)
Comment (benjamineckstein) [reply rest/taskana-rest-spring-example-boot/pom.xml]: removed test classes and test dependencies

File: pom.xml (hunk 3 above)
Comment (benjamineckstein) [reply pom.xml]: Only initialized with the profile coverage activated, but used by surefire plugin. 
IntelliJ shows this as undefined otherwise.

**Pull request merged**